src/routes/HeadlessLogin.jsx

```diff
@@ -0,0 +1,54 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import { logIn } from '../actions/index';
+
+export class HeadlessLogin extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      token: '',
+    };
+  }
+
+  static propTypes = {
+    classes: PropTypes.object.isRequired,
+    dispatch: PropTypes.func.isRequired,
+  };
+
+  handleChange = event => {
+    this.setState({
+      token: event.target.value,
+    });
+  };
+
+  handleLoginButton = () => {
+    const { dispatch } = this.props;
+    const { token } = this.state;
+
+    dispatch(logIn('user.name', 'user.imageUrl', 'user.email', token, 1615064728000));
```

Comment:
So that our tests don't start failing out of the blue next year, let's calculate this dynamically? Something like a day into the future should be good.